package-res/resources/web/prompting/api/EventAPI.js

```diff
@@ -120,5 +120,21 @@ define([], function() {
         api.operation._getPromptPanel().ready = function(){};
       }
     };
+
+    /**
+     * Registers a listener for when the state of the prompt panel or parameter definition changes.
```

Comment:
Probably for users will be useful to describe all state flags which changes affect to stateChanged event. What do you think?